README.md

```diff
@@ -276,10 +276,11 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 
 *Packages that help with building Distributed Systems.*
 
+* [dht](https://github.com/anacrolix/torrent/dht) - BitTorrent Kademlia DHT implementation.
```

Comment:
Also, is this even needed, since you're adding the parent folder https://github.com/anacrolix/torrent below?

If you really want to keep it, please make it a sub-list item. Similar to how `github_flavored_markdown`, a renderer for `blackfriday`, appears as a sub-list item at https://github.com/avelino/awesome-go#text-processing.